lib/Fmt_ast.ml

```diff
@@ -400,20 +404,23 @@ let docstring_epi ~standalone ~next ~epi ~floating =
       str "\n" $ epi
   | _ -> epi
 
-let fmt_docstring c ?(standalone = false) ?pro ?epi doc =
-  list_pn (Option.value ~default:[] doc)
-    (fun ~prev:_ ({txt; loc}, floating) ~next ->
+let fmt_docstring' c ?(standalone = false) ?pro ?epi doc =
```

Comment:
These changes to `fmt_docstring` and the addition of `extract_doc_attrs` were reverted in #2451.